test/greet.test.js

```diff
@@ -0,0 +1,17 @@
+'use strict';
+
+const assert = require('assert');
+const greet = require('../lib/greet');
+
+const name = 'steven';
+
+describe('greet.js', function(){
+  describe('#greet', function(){
```

Comment:
#methodName is a great way to distinguish an instance of a particular method. nice catch.